### ISSUE TITLE
DAAN: Question on computing "w" 

### ISSUE THREAD

**Issue by zjjslch** [jindongwang/transferlearning#339]
1. On DAAN "train.py" line 139, the code first calculates the "global loss" by a factor of 0.05, and then uses it to get the global A-distance (line 142). However, when calculating the local A-distance (line 134), there is no a factor like 0.01 (on line 140). This may result the big difference between the local distance and global distance, and also may lead to the "w" always close to or even larger than 1. What's the purpose of the factor of 0.05? or it is a  bug? 
![image](https://user-images.githubusercontent.com/85175585/176580869-a44b1bfb-3355-471e-a321-a4d4de477b60.png)

2. On DAAN "train.py" line 102, the MU formular is not same as paper. It is 1-"w" in paper, why?
![image](https://user-images.githubusercontent.com/85175585/176583180-19c9876a-06ff-4380-a198-df3e9809c6b3.png)

**Comment by jindongwang** [comment]
Thanks for your information.
1. 0.05 is just a hyperparameter to make the training stable. You can treat it like a trick.
2. The paper mixed the mu with 1-mu. The code is right.
For a complete and more recent version of the DAAN code, please see here: https://github.com/jindongwang/transferlearning/tree/master/code/DeepDA

**Comment by zjjslch** [comment]
Thanks for the second question's answer. For the first Question, I am still confused. 
![image](https://user-images.githubusercontent.com/85175585/177073358-e3f16d8c-a36a-4627-a7f9-8ea0d47ed999.png)
I think on line  142 Using (err_s_domain + err_t_domain) instead global loss will be easier to understand, since global loss has multiplied hyperparameter 0.05, which will lead to large difference between d_c and d_m. Or you just do it on purpose to make global loss much smaller.

**Comment by jindongwang** [comment]
Yes, we just do it to make the loss more comparative...
